zkevm/tests/integration.rs

```diff
@@ -0,0 +1,42 @@
+use halo2_proofs::plonk::{ProvingKey, VerifyingKey};
+use halo2_proofs::poly::commitment::Params;
+use pairing::bn256::G1Affine;
+use rand_xorshift::XorShiftRng;
+use types::eth::test::mock_block_result;
+use zkevm::keygen::{gen_evm_pk, gen_evm_vk, gen_state_pk, gen_state_vk};
+use zkevm::prover::Prover;
+use zkevm::utils::{load_or_create_params, load_or_create_rng};
+use zkevm::verifier::Verifier;
+
+const PARAMS_PATH: &str = "./test_params";
+const RNG_PATH: &str = "./test_rng.json";
```

Comment:
```suggestion
const SEED_PATH: &str = "./test_seed";
```

1. it's actuallt the seed file
2. why json?

Reply:
Oh, sorry. This `json` name was auto-generated by `github compilot`.  I forgot fixing.